aria-practices.html

```diff
@@ -6737,6 +6734,139 @@ <h3>Range properties with spin buttons</h3>
+  ...
+&lt;/svg&gt;
+</code></pre>
+
+      <p class="note">
+        Note that <code>aria-level</code> values greater than 6 for headings are not currently supported by all assistive technologies.
+      </p>
+
+      <section id="aria-level_heading_role_remediation">
+        <h4>Remediation Use Cases for Headings</h4>
+        <p>
+          In rare scenarios, legacy code cannot be converted to use HTML heading elements.
+          In these cases, the <code>aria-level</code> property and the <code>heading</code> role can be added to elements that assistive technologies should treat like native HTML heading elements.
+        </p>
+        <div class="example">
+          <p>
+            For example, consider the following markup as the starting point.
+            It does not follow best practice and is not accessible.
+            We assume in this example the constraint that elements and class names can't be changed, but ARIA attributes can be added.
+          </p>
+
+          <pre><code>&lt;div class="heading-big"&gt;Housing Specification&lt;/div&gt;
+&lt;div id="room-definition" class="heading-small"&gt;Definition of a Room&lt;/div&gt;
+...</code></pre>
+
+          <p>
+            By adding the <code>role</code> and <code>aria-level</code> attributes, we specify that "Definition of a Room" is the heading of a subsection of the section titled "Housing Specification":
+          </p>
+
+          <pre><code>&lt;div role="heading" aria-level="1" class="heading-big"&gt;Housing Specification&lt;/div&gt;
+&lt;div role="heading" aria-level="2" id="room-definition" class="heading-small"&gt;Definition of a Room&lt;/div&gt;
+...
+</code></pre>
+
+          <p>
+            If the original markup <em>could</em> be changed arbitrarily, the equivalent and preferred HTML would be:
+          </p>
+
+          <pre><code>&lt;h1&gt;Housing Specification&lt;/h1&gt;
+&lt;h2 id="room-definition"&gt;Definition of a Room&lt;/h2&gt;
+...
+</code></pre>
+        </div>
+      </section>
+    </section>
+
+    <section id="aria-level_listitem_role">
+      <h3>Using <code>aria-level</code> with the <code>listitem</code> Role</h3>
+      <p>
+        Browsers automatically calculate the value of <code>aria-level</code> for elements with the <code>listitem</code> role.
+        While ARIA 1.2 supports <code>aria-level</code> on <code>listitem</code> elements, the Authoring Practices Task Force has not found a scenario where applying <code>aria-level</code> to a <code>listitem</code> is needed and has
+        <a href="https://github.com/w3c/aria/issues/1369">raised an issue</a>
+        suggesting the support be deprecated.
+      </p>
+    </section>
+
+    <section id="aria-level_treeitem_role">
+      <h3>Using <code>aria-level</code> with the <code>treeitem</code> Role</h3>
+      <p>
+        As with <code>listitem</code> elements, browsers automatically compute an <code>aria-level</code> value for elements with role <code>treeitem</code>.
```

Comment:
Let's not refer to listitem levels in the treeitem section.

```suggestion
        Browsers automatically compute an <code>aria-level</code> value for elements with role <code>treeitem</code>.
```